src/common/k8s-api/endpoints/helm-charts.api.ts

```diff
@@ -324,7 +324,11 @@ export class HelmChart {
   }
 
   getId(): string {
-    return `${this.repo}:${this.apiVersion}/${this.name}@${this.getAppVersion()}+${this.digest}`;
+    const digestPart = this.digest
+      ? `+${this.digest}`
+      : "";
+
+    return `${this.repo}:${this.apiVersion}/${this.name}@${this.getAppVersion()}${digestPart}`;
```

Comment:
Does this risk the id not being unique (and would that be a problem)?

Reply:
Given that we also have the `repo` in there I don't think that the IDs won't be unique anymore.